### PR TITLE
docs: DTS

### PR DESCRIPTION
**Pull request by Timeless0911**
## Summary

- [x] dts in advanced
- [x] lib.dts config
- [x] rsbuild-plugin-dts readme doc

## Checklist

- [ ] Tests updated (or not required).
- [ ] Documentation updated (or not required).

### CODE REVIEW

File: packages/plugin-dts/README.md [hunk 1]
```diff
@@ -2,18 +2,169 @@
...
+  distPath: './dist-types',
+});
+```
+
+### build
+
+- **Type:** `boolean`
+- **Default:** `false`
+
+Determines whether to generate DTS files while building the project's references. This is equivalent to using the `--build` flag with the `tsc` command.
```
Comment (fi3ework) [inline packages/plugin-dts/README.md:78]:
```suggestion
Determines whether to generate DTS files while building the project references. This is equivalent to using the `--build` flag with the `tsc` command.
```

Comment (fi3ework) [reply packages/plugin-dts/README.md:78]: If it's set to `false`, it won't generate DTS files, right?

File: packages/plugin-dts/README.md [hunk 2]
```diff
@@ -2,18 +2,169 @@
...
+### abortOnError
+
+- **Type:** `boolean`
+- **Default:** `true`
+
+Whether to abort the build process when an error occurs during DTS generation.
+
+By default, type errors will cause the build to fail. When `abortOnError` is set to `false`, the build will still succeed even if there are type issues in the code.
+
+```js
```
Comment (fi3ework) [inline packages/plugin-dts/README.md:91]: Adding ``For example, disable `abortOnError` to allow type error.``. The following fields also need to update.

File: packages/plugin-dts/README.md [hunk 3]
```diff
@@ -2,18 +2,169 @@
...
+- **Type:** `string`
+- **Default:** `undefined`
+
+```js
+pluginDts({
+  footer: '/** @footer */',
+});
+```
+
+Inject content into the bottom of each DTS file.
```
Comment (fi3ework) [inline packages/plugin-dts/README.md:162]: examaple is above description.

File: packages/plugin-dts/README.md [hunk 4]
```diff
@@ -2,18 +2,169 @@
...
 
-https://lib.rsbuild.dev/
+Read [DTS](https://lib.rsbuild.dev/guide/advanced/dts) and [lib.dts](https://lib.rsbuild.dev/config/lib/dts) for more details.
+
+## Using in Rsbuild
+
+Install:
+
+```bash
+npm add rsbuild-plugin-dts -D
```
[the remark this answers is not in the document]
Comment (Timeless0911) [reply packages/plugin-dts/README.md:18]: This is readme.

File: packages/plugin-dts/README.md (hunk 1 above)
Comment (Timeless0911) [reply packages/plugin-dts/README.md:78]: This controls whether to build dependencies of project references, the dts file generation can only be disabled by `dts: false` or do not have dts configurations.

**Review by fi3ework — APPROVED** [review]

**Pull request merged**